CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -42,13 +42,14 @@ The main contribution of this book is the "Legacy Code Change Algorithm":
 3. **Add new functionality with new tests** (usually following the Test Driven Development (TDD) process).
 4. **Refactor** tested code to remove duplication, clean up, etc.
 
-Above, Step 1 defines the targeted subset of legacy code that must have tests which depends on what needs to change, what can be sensed, and where dependencies can be broken.
-After the targeted legacy code has been covered with unit tests in Step 2, it should have close to 100% coverage (otherwise one can not claim that existing behavior is being preserved, unless the uncovered code and behavior is never used by anyone, in which case it should likely be removed).
-Any refactorings in Step 4 are safe to perform once the targeted code is sufficiently covered with tests.
+Step 1 defines the targeted subset of legacy code that must have tests which depend on what needs to change, what can be sensed, and where dependencies can be broken.
+After unit tests which cover the targeted code have been implemented in Step 2, it should have close to 100% coverage.
+Without full coverage, it is difficult to know if existing behavior is preserved.
+Any refactoring in Step 4 is safe to perform once the targeted code is sufficiently covered with tests.
```

Comment:
Ok, fair enough. I had never seen it written this way before but that is fine.

Reply:
So, just curious but which of us is responsible for hitting the **Resolve conversation** button for these? Me or you?

Reply:
@markcmiller86 asked>

> So, just curious but which of us is responsible for hitting the **Resolve conversation** button for these? Me or you?

Not sure what the convention is for this but since I started the conversation in a review, I guess I should be the way to resolve it.  I guess you can think of one of these reviews and the conversation that follows as a subtask in the larger PR code review.  So to resolve the conversation is to resolve the issues raised in the review?